test/e2e/openLibraryLogo.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { test, expect, Page } from "@playwright/test";
+import { OpenLibraryLogo } from "../pages/OpenLibraryLogo";
+import { BASE_URL } from "../helper-config";
+
+let openLibraryLogoPage: OpenLibraryLogo;
+
+test.beforeEach(async({ page }: {page: Page}) => {
+    openLibraryLogoPage = new OpenLibraryLogo({page});
+    await openLibraryLogoPage.navigateBasePage();
```

Comment:
IMHO, this sounds better &darr;
```suggestion
    await openLibraryLogoPage.navigateToHome();
```